app/src/main/java/me/ccrama/redditslide/SettingValues.java

```diff
@@ -468,8 +470,17 @@ public static TimePeriod getSubmissionTimePeriod(String sub) {
 
 
     public static boolean isNight() {
-        int hour = Calendar.getInstance().get(Calendar.HOUR_OF_DAY);
-        return (hour >= nightStart + 12 || hour < nightEnd) && tabletUI && nightMode;
```

Comment:
I removed the check for `tabletUI`. It felt wrong, but if it is not, let me know and I can put it back.